source/crud/crud.rst

```diff
@@ -255,7 +255,18 @@ Read
      * @see https://docs.mongodb.com/manual/reference/command/aggregate/
      */
     maxTimeMS: Optional<Int64>;
-    
+
+    /**
+     * The maximum amount of time for the server to wait on new documents to satisfy a tailable cursor
+     * query. This only applies to aggregations which return a TAILABLE_AWAIT cursor. Drivers
+     * SHOULD NOT send this value when the cursor is not a TAILABLE_AWAIT cursor, but if they do
+     * the server will ignore it.
```

Comment:
> but if they do the server will ignore it."

Is this true? I would not be surprised if the server considers this undefined (untested) behavior that they are free to change between releases.